source/includes/lead_event_feed_service/call_events/_destroy.md

```diff
@@ -0,0 +1,52 @@
+### Delete a Call Event
+
+```ruby
+require 'uri'
+require 'net/http'
+
+url = URI("https://api.reachlocalservices.com/leads/call_events/123")
+
+http = Net::HTTP.new(url.host, url.port)
+
+request = Net::HTTP::Delete.new(url)
+request["Authorization"] = 'token lambda_token'
+request["email"] = 'EMAIL_ADDRESS'
```

Comment:
I know that the email header isn't required, but is it even used by LEFS?

If it's used, can we add some explanation about what this is?  Who's email address is it.  I would hate to see a client actually use EMAIL_ADDRESS as the value (trust me, someone will actually do that)

Reply:
The email header is used and necessary. LEFSe uses it when connecting to identity service. In the case of the API gateway, it's a little weird as the email is no longer a parameter, but rather the email address used for gateway authentication.